leaflet-fullscreen/leaflet-fullscreen.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for Leaflet.fullscreen v1.3.0
+// Project: https://github.com/brunob/leaflet.fullscreen
```

Comment:
I guess Project is `https://github.com/Leaflet/Leaflet.fullscreen` rather than this.